windows_create_package.bat

```diff
@@ -35,6 +35,15 @@ if !errorlevel! neq 0 exit /b !errorlevel!
 copy C:\%output_user_root%\openvino\runtime\bin\intel64\Release\*.dll dist\windows\ovms
 if !errorlevel! neq 0 exit /b !errorlevel!
 
+:: Prepare self-contained python
```

Comment:
Is line 29 copy  %cd%\bazel-out\x64_windows-opt\bin\src\python39.dll dist\windows\ovms

Still required?